package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
```

Comment:
Not that important since we aren't publishing this for any production use, but I'd typically think it should be a "devDependencies" entry.

Reply:
shows my lack of experience outside of the clojure side of things, updated here:
https://github.com/cerner/clara-rules/pull/491/commits/e0baa0547673355230cb7d655e34b17b59a412c3